client/src/pages/Doctor_details/Doctor_details.js

```diff
@@ -0,0 +1,255 @@
+import React, { Component } from "react";
+import "./Doctor_details.css";
+class DoctorDetails extends Component {
+  state = {
+    clinic_name: "",
+    speciality: "",
+    experience: "",
+    qualification: "",
+    fees: "",
+    timings: "",
+    gender: "",
+    address: "",
+    city: "",
+    state_input: "",
+    pincode: "",
+    website: "",
+    contact: "",
+  };
+  handleChange = (e) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState({ [name]: value });
+  };
+
+  postData = (e) => {
+    e.preventDefault();
+    fetch("http://localhost:8080/feed/post", {
+      method: "post",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: "Bearer " + localStorage.getItem("jwt"),
```

Comment:
correct the minor typo mistake too. add the inverted commas around Authorization